HW17/src/calculator.ts

```diff
@@ -0,0 +1,24 @@
+export class Calculator {
+    constructor() {};
+    public numbersAdder(firstNumber: number, secondNumber: number, operation: '+') {
+        if (operation == '+') {
+            return firstNumber + secondNumber;  
+        }
```

Comment:
Why do you need the third parameter here if your mehod is called `adder`?
It should always return the sum of two parameters passed. The same is for 3 other methods. Please rewrite them.

Reply:
Corrections made,
thank you!